datadog/resource_datadog_synthetics_test_.go

```diff
@@ -203,7 +203,8 @@ func syntheticsTestOptions() *schema.Schema {
 }
 
 func resourceDatadogSyntheticsTestCreate(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*datadog.Client)
+	providerConf := meta.(*ProviderConfiguration)
+	client := providerConf.CommunityClient
```

Comment:
Nit: Should we print the same error messages in the create/update calls this resource makes?

Reply:
Good Catch!!